src/test_workflow/README.md

```diff
@@ -237,6 +248,9 @@ After the performance test completes, it will report back the test results as we
 
 The development is tracked by meta issue [#126](https://github.com/opensearch-project/opensearch-build/issues/126)
 
+#### benchmarkTest job
```

Comment:
Capitalize Like Other Titles

You can remove this for now since we don't have content.

Reply:
Removed for now. Will add once we have the job ready to be merged.